src/styles/main.scss

```diff
@@ -111,6 +109,9 @@
 // BREAKPOINT 1:
 // Screen is big enough to display the text snippets
 @media (min-width: $breakpoint-medium) {
+  .aa-dropdown-menu {
+    min-width: 400px;
```

Comment:
Yes maybe, or just putting 400px in a variable defined at the top so it feels a bit less magic (and overridable by users).